corehq/apps/registry/utils.py

```diff
@@ -237,9 +242,21 @@ def action_options(is_owner):
         ]
 
 
-def get_data_registry_dropdown_options(domain, required_case_types=None):
+def get_data_registry_dropdown_options(domain, required_case_types=None, permission_checker=None):
+    all_registries = DataRegistry.objects.visible_to_domain(domain)
+    registries = []
+    if permission_checker:
```

Comment:
will this ever be None?

Reply:
Ope added that fix here: [0eb2246](https://github.com/dimagi/commcare-hq/pull/31067/commits/0eb224667ebce8a59d08c85668792b987993d47b) (thanks for the bump)

Reply:
@stephherbers in what context is the permission checker not provided? Won't that bypass the security checks?

Reply:
Yes, it would be the same functionality as it was before. I added the permission checker to all instances in this reports permissions update where we want checks, but here's an example where I thought the checks would not be desired for the "Roles and Permissions" Page: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/views/__init__.py#L697. Does that match your understanding @snopoke?

Reply:
right ok, that makes sense